### ISSUE TITLE
Using alt, loop or par in certain sequence diagrams generate excessive whitespace

### ISSUE THREAD

**Issue by VictorPato** [hikerpig/pintora#153]
## Desired behavior
While using the `alt`, `loop` or `par` keywords in flow diagrams, the content of the generated boxes fit correctly.

## Actual behavior
If an `alt`, `loop` or `par` blocks contain some components (self arrows seem to make it worse?), the generated box contains excessive whitespace. 

### Code
The following Pintora diagram (`input.pintora`) displays the problem:
```
sequenceDiagram
  participant [<box> service_1] as "Service 1"
  participant [<box> service_2] as "Service 2"
  participant [<box> service_3] as "Service 3"
  participant [<box> service_4] as "Service 4"
  service_1->>+service_2: req 0.........
  == some divider ==
  par #ffffff one
    service_2->>+service_3: .........req 1...........
    service_3->>+service_4: .........req 2...........
    service_4->>service_4: req 4
    service_4->>-service_3: req 3
    service_3->>-service_2: req 5
    service_2->>service_2: req 6
  end
  service_2->>-service_1: 
```
### Commands
```
➜  documentation-pintora node --version
v14.15.0
➜  documentation-pintora pintora --version
0.6.1
➜  documentation-pintora pintora render --input input.pintora --output output.png
✔ Render success, saved to output.png         
```
### Output
![output](https://user-images.githubusercontent.com/20075551/192316017-73b6da00-8c2c-4395-a2cd-047f702d0f11.png)
### Live editor
[Link](https://pintorajs.vercel.app/demo/live-editor/?code=c2VxdWVuY2VEaWFncmFtJTBBJTIwJTIwcGFydGljaXBhbnQlMjAlNUIlM0Nib3glM0UlMjBzZXJ2aWNlXzElNUQlMjBhcyUyMCUyMlNlcnZpY2UlMjAxJTIyJTBBJTIwJTIwcGFydGljaXBhbnQlMjAlNUIlM0Nib3glM0UlMjBzZXJ2aWNlXzIlNUQlMjBhcyUyMCUyMlNlcnZpY2UlMjAyJTIyJTBBJTIwJTIwcGFydGljaXBhbnQlMjAlNUIlM0Nib3glM0UlMjBzZXJ2aWNlXzMlNUQlMjBhcyUyMCUyMlNlcnZpY2UlMjAzJTIyJTBBJTIwJTIwcGFydGljaXBhbnQlMjAlNUIlM0Nib3glM0UlMjBzZXJ2aWNlXzQlNUQlMjBhcyUyMCUyMlNlcnZpY2UlMjA0JTIyJTBBJTIwJTIwc2VydmljZV8xLSUzRSUzRStzZXJ2aWNlXzIlM0ElMjByZXElMjAwLi4uLi4uLi4uJTBBJTIwJTIwJTNEJTNEJTIwc29tZSUyMGRpdmlkZXIlMjAlM0QlM0QlMEElMjAlMjBwYXIlMjAlMjNmZmZmZmYlMjBvbmUlMEElMjAlMjAlMjAlMjBzZXJ2aWNlXzItJTNFJTNFK3NlcnZpY2VfMyUzQSUyMC4uLi4uLi4uLnJlcSUyMDEuLi4uLi4uLi4uLiUwQSUyMCUyMCUyMCUyMHNlcnZpY2VfMy0lM0UlM0Urc2VydmljZV80JTNBJTIwLi4uLi4uLi4ucmVxJTIwMi4uLi4uLi4uLi4uJTBBJTIwJTIwJTIwJTIwc2VydmljZV80LSUzRSUzRXNlcnZpY2VfNCUzQSUyMHJlcSUyMDQlMEElMjAlMjAlMjAlMjBzZXJ2aWNlXzQtJTNFJTNFLXNlcnZpY2VfMyUzQSUyMHJlcSUyMDMlMEElMjAlMjAlMjAlMjBzZXJ2aWNlXzMtJTNFJTNFLXNlcnZpY2VfMiUzQSUyMHJlcSUyMDUlMEElMjAlMjAlMjAlMjBzZXJ2aWNlXzItJTNFJTNFc2VydmljZV8yJTNBJTIwcmVxJTIwNiUwQSUyMCUyMGVuZCUwQSUyMCUyMHNlcnZpY2VfMi0lM0UlM0Utc2VydmljZV8xJTNBJTIw&config=JTdCJTIydGhlbWVDb25maWclMjIlM0ElN0IlMjJ0aGVtZSUyMiUzQSUyMmRhcmslMjIlN0QlMkMlMjJzZXF1ZW5jZSUyMiUzQSU3QiU3RCUyQyUyMmVyJTIyJTNBJTdCJTdEJTJDJTIyY29tcG9uZW50JTIyJTNBJTdCJTdEJTJDJTIyYWN0aXZpdHklMjIlM0ElN0IlN0QlMkMlMjJtaW5kbWFwJTIyJTNBJTdCJTdEJTdE)

**Comment by hikerpig** [comment]
@VictorPato I've fixed this in the latest master branch but not released yet, I will release the next patch version in this week along with some tiny features.

By now the live editor has already updated, but it seems like you use the cli and vscode extension more, so please wait several days for the next release.

**Comment by VictorPato** [comment]
@hikerpig It's looking great on the live editor. Thank you for the fix and for the quick response! 🏆 
You can close this issue as you see fit.

**Comment by hikerpig** [comment]
Released in v0.6.2
